src/annotator/guest.js

```diff
@@ -538,41 +521,6 @@ export default class Guest extends Delegator {
     return this.createAnnotation({ $highlight: true });
   }
 
-  // Create a blank comment (AKA "page note")
-  createComment() {
-    const annotation = {};
-
-    const prepare = info => {
-      annotation.document = info.metadata;
-      annotation.uri = info.uri;
-      annotation.target = [{ source: info.uri }];
-    };
-
-    this.getDocumentInfo()
-      .then(prepare)
-      .then(() => this.publish('beforeAnnotationCreated', [annotation]));
-
-    return annotation;
-  }
-
-  // Public: Deletes the annotation by removing the highlight from the DOM.
-  // Publishes the 'annotationDeleted' event on completion.
-  //
-  // annotation - An annotation Object to delete.
-  //
-  // Returns deleted annotation.
-  deleteAnnotation(annotation) {
```

Comment:
This code was never called in the app. Annotations are always deleted from the sidebar rather than the annotator side of the app. This in turn triggers a call to the `detach` method in this class.